### DISCUSSION TITLE
If you are certain that you will never need ssr, would you still use razzle as a better cra?

### DISCUSSION THREAD

**Discussion by ghost** [jaredpalmer/razzle#1708] [Q&A] [answered]
Cra has more community contributions, but razzle seems to have a lot of activity lately.

Some things that I use or may start to use later:
- rescript
- react
- emotion
- rescript react router
- vite
- esbuild / swc

**Comment by fivethreeo** [comment, marked as answer]
The advantage is that razzle can be configured without breaking out.

But if cra works and you have no need for custom configuration there is no need for razzle.

**Reply by ghost** [reply]
How about the choices within the babel and webpack configs? How general are the default decisions for most applications? Is create-react-app or razzle more opinionated?

**Comment by fivethreeo** [comment]
Babel config is mostly like next, webpack is with mini-css-extract and file/url loader. Pretty basic and mostly unopinionated.
